data/mapcss-mapping.csv

```diff
@@ -1683,3 +1683,4 @@ railway|station|subway|singapore;[railway=station][transport=subway][city=singap
 railway|subway_entrance|singapore;[railway=subway_entrance][city=singapore];;name;int_name;1595;
 railway|station|subway|boston;[railway=station][transport=subway][city=boston],[railway=station][station=subway][city=boston];;name;int_name;1596;
 railway|subway_entrance|boston;[railway=subway_entrance][city=boston];;name;int_name;1597;
+leisure|golf_course|miniature;[leisure=miniature_golf];;name;int_name;1598;
```

Comment:
why not `leisure|miniature_golf;...`?
I think better keep it closer to the original OSM tagging then try to group

Reply:
e.g. I'm pretty sure our Editor implementation reconstructs original OSM tags from the type name, hence it won't be able to derive "leisure=miniature_golf" from "leisure-golf_course-miniature"..

Reply:
Yup, i did fix that but looks like i forgot to push..

Will be able to fix review later today/tomorrow 👍